src/components/jobs/JobItem.jsx

```diff
@@ -1,31 +1,41 @@
 import React from 'react';
 import styled from 'styled-components';
-import { Card, Image } from 'semantic-ui-react';
+import { Icon, Image, Label } from 'semantic-ui-react';
+import StyledCard from '../atoms/Card';
 
 const Job = props => (
   <JobWrapper>
-    <Card fluid className="card" href={'/vacatures/' + props.job.slug}>
-      <Card.Content className="content-container">
+    <StyledCard className="card">
+      <Label
+        color={props.job.isJob ? 'blue' : 'yellow'}
+        ribbon="right"
+        className="ribbon"
+      >
+        <Icon
+          name={props.job.isJob ? 'briefcase' : 'student'}
+          size="large"
+          className="ribbon-icon"
+        />
+        {console.log(props.job.type)}
```

Comment:
You left your debugging code in!

Reply:
Well I thought I didn't, but it seems by switching branches something went wrong :p... I will remove this ofcourse!

Reply:
Fix incoming